src/screens/Login/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import Container from 'components/Container';
+import Button from 'components/Button';
+
+function Login() {
+  const { t, i18n } = useTranslation();
+
+  return (
+    <Container>
+      <Link to="/sign_up">
+        <Button label={t('Login:lblSignUp')} loading={false} />
```

Comment:
This button is not necessary, just leave the Link component

Reply:
The button component has been removed :suspect: